lib/card.rb

```diff
@@ -38,4 +38,25 @@ class Card < ActiveRecord::Base
   TRACKED_FIELDS = %w(name type_id db_content trash)
   Loader.load_mods if count > 0
 
+
+  def self.find_each(options = {})
+    if block_given?
+      find_in_batches(options) do |records|
+        records.each { |record| yield record }
+      end
+    else
+      super(options)
+    end
+  end
```

Comment:
fwiw, in the master branch it's actually moved out of card.rb and into a ClassMethods module somewhere (collection.rb, I think), so it was a slightly different loading situation when I saw it pass.